### PR TITLE
Create CODEOWNERS

### PR DESCRIPTION
**Pull request by lpil**
We are making our open source project maintainers explicit with a CODEOWNERS file. Let me know if these are not the correct maintainers for this project

### Checklist

* [x] I have read the [Contributor Guide](../../CONTRIBUTING.md)
* [x] I have read and agree to the [Code of Conduct](../../CODE_OF_CONDUCT.md)
* [x] I have added a description of my changes and why I'd like them included in the section below

### CODE REVIEW

**Comment by repi** [comment]
@Jake-Shadle should we move this to just be you that maintain it instead of me? It is really part of cargo-deny and the only thing complex in it is how to upgrade it when cargo-deny is upgraded

**Comment by Jake-Shadle** [comment]
Yah that would be fine.

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+* @repi
```
Comment (repi) [inline .github/CODEOWNERS:1]:
```suggestion
* @JakeShadle
```

**Review by repi — CHANGES_REQUESTED** [review]

**Comment by lpil** [comment]
Thanks gang

**Review by repi — APPROVED** [review]

**Pull request merged**